tests/labhub_test.py

```diff
@@ -88,22 +88,26 @@ def test_create_issue_cmd(self):
         plugins.labhub.GitHubToken.assert_called_with(None)
         plugins.labhub.GitLabPrivateToken.assert_called_with(None)
 
-
         labhub.REPOS = {'repository': self.mock_repo,
                         'repository.github.io': self.mock_repo}
 
         testbot.assertCommand('!new issue repository this is the title\nbo\ndy',
                               'Here you go')
 
         labhub.REPOS['repository'].create_issue.assert_called_once_with(
-            'this is the title', 'bo\ndy\nOpened by @None at [text]()'
+            'this is the title', 'bo\ndy\nOpened on text'
         )
 
-        testbot.assertCommand('!new issue repository.github.io another title\nand body',
-                              'Here you go')
+        msg = Message('!new issue repository.github.io another title\nand body',
+                      'Here you go')
+        msg.extras['url'] = 'http://example.com'
+        msg.frm = testbot.bot.build_identifier('Jo')
+        msg.frm._nick = 'jo'
```

Comment:
`msg.frm._nick` --> `msg.frm.nick`

Reply:
I am pretty sure it is a non-writable property.